operator/src/test/java/org/bf2/operator/events/ControllerEventFilterTest.java

```diff
@@ -27,7 +27,7 @@ void setup() {
 
     @Test
     void testAddResourceAccepted() {
-        assertTrue(target.acceptChange(mockConfiguration, null, new ManagedKafka()));
+        assertTrue(target.accept(null, new ManagedKafka()));
```

Comment:
This test method and the `mockConfiguration` field and code in `setup` can also be removed. We no longer need to test the `add` case because the filter is only for updates.

Reply:
Sure. I'll update this.